libs/components/phone-field/src/lib/modules/phone-field/phone-field-input.directive.ts

```diff
@@ -119,8 +119,15 @@ export class SkyPhoneFieldInputDirective
       .subscribe(() => {
         const value = this.#adapterSvc?.getInputValue(this.#elRef);
         this.#setValue(value);
-        this.#notifyChange?.(this.#getValue());
-        this.#notifyTouched?.();
+        this.#control?.updateValueAndValidity();
+      });
+
+    this.#phoneFieldComponent.countrySearchForm
+      .get('countrySearch')
+      ?.valueChanges.pipe(takeUntil(this.#ngUnsubscribe), take(1))
```

Comment:
Do we need both `takeUntil` and `take`?

Reply:
Yes - because this isn't something that is always guaranteed to fire. For example, if a user has a phone field and it defaults to the US and they don't need to change it to another country - this would never fire and we would still want to clean up the subscription when the component is destroyed.